### DISCUSSION TITLE
time remaining on download times

### DISCUSSION THREAD

**Discussion by Xstoudi** [chocolatey/choco#2456] [Q&A]
Hey, can I add an estimation of download time to the Ftp and Web downloaders?

**Comment by pauby** [comment]
@Xstoudi I've converted this to a discussion as it didn't look to be an issue.

**Reply by Xstoudi** [reply]
My bad, I just made what `CONTRIBUTING.md` told me for enhancement requests.
> Log a github issue. There are fewer constraints on this versus reporting issues.
